<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addinfo_student</class>
 <widget class="QMainWindow" name="addinfo_student">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <spacer name="horizontalSpacer_7">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>请选择院系:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <spacer name="horizontalSpacer_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="3">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0">
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="2">
     <widget class="QComboBox" name="comboBox_grade"/>
    </item>
    <item row="1" column="3">
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="3">
     <spacer name="horizontalSpacer_8">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="2">
     <widget class="QComboBox" name="comboBox_depart"/>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>请选择年级:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>请输入姓名:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="lineEdit_numberinput"/>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="lineEdit_nameinput"/>
    </item>
    <item row="0" column="0">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>请输入学号:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>581</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>选择</string>
    </property>
    <addaction name="action_save"/>
    <addaction name="action_exit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_save">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
